pkg/webui/console/api/index.js

```diff
@@ -101,6 +101,7 @@ export default {
     list: ttnClient.Applications.Devices.getAll.bind(ttnClient.Applications.Devices),
     get: ttnClient.Applications.Devices.getById.bind(ttnClient.Applications.Devices),
     create: ttnClient.Applications.Devices.create.bind(ttnClient.Applications.Devices),
+    update: ttnClient.Applications.Devices.updateById.bind(ttnClient.Applications.Devices),
```

Comment:
please see how the `applications` and `application` definitions are structured. lets be consistent